recipes/libmp3lame/all/conanfile.py

```diff
@@ -0,0 +1,110 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+import os
+import shutil
+
+
+class LibMP3LameConan(ConanFile):
+    name = "libmp3lame"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "LAME is a high quality MPEG Audio Layer III (MP3) encoder licensed under the LGPL."
+    homepage = "http://lame.sourceforge.net"
+    topics = ("conan", "libmp3lame", "multimedia", "audio", "mp3", "decoder", "encoding", "decoding")
+    license = "LGPL-2.0"
+    exports_sources = ["6410.patch", "6416.patch", "android.patch"]
```

Comment:
Could you clarify the purpose of those patches?

Reply:
according to https://github.com/bincrafters/conan-libmp3lame/commit/a79d68930ba60bc9e7bf9a874af491854d3f60b7 6410.patch and 6516.patch are needed for mingw
android.patch fixes this issue: https://github.com/bincrafters/community/issues/588

Reply:
@ericLemanissier these numbers refer to SVN commits from their repository (however, not released yet...):
https://sourceforge.net/p/lame/svn/6410/
https://sourceforge.net/p/lame/svn/6416
both commits are about some MinGW specific issues:
https://sourceforge.net/p/lame/bugs/487/
https://sourceforge.net/p/lame/bugs/75/